examples/render/contour.yaml

```diff
@@ -76,6 +76,8 @@ data:
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
+  annotations:
```

Comment:
Should we generate an "experimental" version of the rendered? I don't think the annotations would matter to someone without the flag, but it's additional settings that someone might not need.

Reply:
These annotation are just added by the newer version of controller-gen required for the service-apis. They can't be turned off, it's just because we updated the dependency, sadly.

Reply:
That’s kinda obnoxious but c’est la vie

Reply:
It doesn't do anything, it's added by controller-gen only, not needed for the new code.